ui/component/or-asset-tree/package.json

```diff
@@ -19,6 +19,7 @@
     "@mdi/js": "^5.9.55",
     "@openremote/core": "workspace:*",
     "@openremote/model": "workspace:*",
+    "@openremote/or-asset-viewer": "^1.1.0",
```

Comment:
Don't see any imports from this added package? Would also need to be added to tsconfig.json if there was a dependency

Reply:
I forget to clean this ! It'll be cleared